docs/appendices/release-notes/5.7.2.rst

```diff
@@ -104,7 +104,7 @@ Fixes
   query.
 
 - Fixed an issue that could lead to out-of-memory errors when executing
-  ``lead`` or ``lag`` window functions on a large table.
+  window functions on a large table.
```

Comment:
If this makes it to 5.7.2 - I think it makes sense to put 2 accounting fixes for window functions under same entry? 

lag/lead are also affected by this (for big tables parallel sort should kick-in).